### ISSUE TITLE
fix(alertdialog): missing test for aria-describedby 

### ISSUE THREAD

**Issue by veiko** [veiko/jest-a11y#21]
### Description

As part of the [WAI alert dialog pattern](https://www.w3.org/WAI/ARIA/apg/patterns/alertdialog/), `aria-describedby` is called out as a requirement:
> The element with role alertdialog has a value set for [aria-describedby](https://w3c.github.io/aria/#aria-describedby) that refers to the element containing the alert message.
